### PR TITLE
TEDU-103 review of how to update Mautic

### PR DESCRIPTION
**Pull request by favour-chibueze**
review of how to update Mautic section

<a href="https://gitpod.io/#https://github.com/mautic/user-documentation/pull/90"></a>

### CODE REVIEW

File: docs/getting_started/how_to_update_mautic.rst [hunk 1]
```diff
@@ -13,10 +13,11 @@ There are two ways to update Mautic:
 If your instance is in production, has a large number of Contacts and/or is  on shared hosting, it's **strongly** recommended that you update at the command line.
 
 .. warning::
-    Updating in the User interface requires a significant amount of resources, and can be error-prone if the server restricts resource allocation. This can lead to failed updates and corrupted data. This feature will be completely removed in Mautic 5.0 and you will have to update at the command line.
+    Updating in the User interface requires a significant amount of resources, and can be error-prone if the server restricts resource allocation. A failed update or corrupted data can result from this. This feature would be completely removed in Mautic 5.0 and you have to update at the command line.
```
Comment (RCheesley) [inline docs/getting_started/how_to_update_mautic.rst:16]:
```suggestion
    Updating in the User interface requires a significant amount of resources, and can be error-prone if the server restricts resource allocation. A failed update or corrupted data can result from this. It's planned to remove this feature in Mautic 5.0, requiring updating at the command line.
```

File: docs/getting_started/how_to_update_mautic.rst [hunk 2]
```diff
@@ -29,13 +30,14 @@ Mautic can only be updated using Composer via the command line from version 5.0.
...
   :alt: Screenshot showing deprecated update feature warning
 
 .. warning::
-    Before starting to upgrade, it is highly recommended to take a backup of your instance. If updates are available, you will receive an update notification followed by step-by-step instructions in the CLI to complete the process.
+    Before starting to upgrade, it's highly recommended to take a backup of your instance. If updates are available, you will receive an update notification followed by step-by-step instructions in the command-line interface to complete the process.
 
-Log in via the command line, and change directory to the Mautic directory using the command
+Log in via the command line, and change directory to the Mautic directory using the command:
 
 ``cd /your/mautic/directory``
```
Comment (RCheesley) [inline docs/getting_started/how_to_update_mautic.rst:42]: Should we use the proper code-block here? Or is there value in just using the inline code markup?

**Review by RCheesley — COMMENTED** [review]
Some suggestions to review.

Also we have not got any mention here about updating with composer-based installation, which I believe we were planning to integrate into this section and remove it from the composer installation section (ideally just linking back to the correct heading on this page).

Please add it to the PR so that this covers all ways of updating Mautic, and you can also remove it from the composer page in the same PR, replacing it with the correct link.

File: docs/getting_started/how_to_update_mautic.rst (hunk 2 above)
Comment (favour-chibueze) [reply docs/getting_started/how_to_update_mautic.rst:42]: inline code markup works

**Review by RCheesley — APPROVED** [review]
Good to go! 🥳

**Pull request merged**